packages/local_auth/local_auth_windows/windows/local_auth_plugin.cpp

```diff
@@ -0,0 +1,223 @@
+// Copyright 2013 The Flutter Authors. All rights reserved.
+// Use of this source code is governed by a BSD-style license that can be
+// found in the LICENSE file.
+#include "local_auth.h"
+
+namespace local_auth_windows {
+
+using namespace flutter;
+using namespace winrt;
+
+template <typename T>
+// Helper method for getting an argument from an EncodableValue
+T GetArgument(const std::string arg, const EncodableValue* args, T fallback) {
```

Comment:
All of these private helpers and classes should be in an anonymous namespace to give them internal linkage.

Reply:
Done.